src/interfaces/IWell.sol

```diff
@@ -16,6 +16,17 @@ struct Call {
  * @title IWell is the interface for the Well contract.
  */
 interface IWell {
+    /// @notice Returns the slippage of swapping `amountOut` of `fromToken` for `toToken`.
+    /// @dev Returns only a uint amountOut and a uint minAmountOut.
+    error SlippageOut(uint amountOut, uint minAmountOut);
+    /// @notice Returns the slippage of swapping `amountIn` of `fromToken` for `toToken`.
+    /// @dev Returns only a uint amounntIn and a uint maxAmountIn.
+    error SlippageIn(uint amountIn, uint maxAmountIn);
+    /// @notice Returns the amount of leaves the tree has.
```

Comment:
Is this ChatGPT? Let's update this to describe what actually happens. Refer to the `_getIJ` function.

Reply:
No just human error, I'll get right on this.